src/components/graph.tsx

```diff
@@ -82,24 +86,31 @@ const possibleYlabels = [
   "Temperature",
   "Speed",
   "Frequency",
+  "numberOfCustomers",
   "Density",
   "UnitsSold",
   "Satisfaction",
 ];
 
 export function extractGraphData(data: string): GraphData | null {
-  console.log("extracting graph data", data);
   try {
     data = JSON.parse(data);
   } catch {
-    console.log("this be null");
+    console.log("Could not parse data: ${data} as json");
```

Comment:
Need to change quotes to these bad boys:

```
`
```

Reply:
good spot